### ISSUE TITLE
Accessibility, Section 508 Compliance: On the APIs page, A user is able to tab into the Search field. However a user is unable to tab out of the search field. 

### ISSUE THREAD

**Issue by christopherwoodland** [Azure/api-management-developer-portal#1404]
@MikeBazMSFT Budzynski

Our client has noticed the following accessibility concern. This issue is an extension of the previous issues entered pertaining to 508 compliances (#1374, #1375, #1378, #1379, #1380.), which the customer is bound to. I am willing to aid in any way to bring these concerns to resolution. Please let me know if additional context or information is required to help move these concerns along. I can be reached internally via teams or email cwoodland@microsoft.com
Thank you in advance!

1.	On the APIs page, a user can tab into the Search field, however a user is unable to tab out of the search field.

 
![image](https://user-images.githubusercontent.com/16299417/127706507-755c4a81-7e11-4e34-b2e9-d08f585d6452.png)

**Comment by MikeBazMSFT** [comment]
I do not understand why I was tagged here - I'm guessing a typo and that @mikebudzynski was intended.

**Comment by mikebudzynski** [comment]
@MikeBazMSFT, it's likely a mistake, please disregard.
